client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -561,25 +562,37 @@ const Editor = (props: EditorProps): JSX.Element => {
     return domNode;
   }
 
+  function clearTestFeedback() {
+    const testStatus = document.getElementById('test-status');
+    if (testStatus) {
+      testStatus.innerHTML = '';
+    }
+  }
```

Comment:
```suggestion
  function clearTestFeedback() {
    const testStatus = document.getElementById('test-status');
    if (testStatus && testStatus.innerHTML) {
      const currentHeight = `${testStatus.offsetHeight}px`;
      testStatus.style.height = currentHeight;
      testStatus.innerHTML = '';
    }
  }

  function clearTestStatusHeight() {
    const testStatus = document.getElementById('test-status');
    if (testStatus) testStatus.style.height = '';
  }
```

Reply:
@ojeytonwilliams I left the changing box size in there because I thought it would make it more obvious that the message had indeed refreshed (i.e. that the check your code button worked) if the exact same message was displayed again, but I can see your point and since the text does disappear briefly I think that is probably enough of an indicator. I added your changes to the `clearTestFeedback` function but I don't thing we need the `clearTestStatusHeight` function. Rather, I just cleared the height on the status div after the status message is displayed. I needed to do this because when the status message changes the height of that div needs to change as well and removing the height and letting the box size naturally based on the content does the trick.

Reply:
Yeah, that works too. As long as we clear the height before the editor resizes, we're good.